packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -397,7 +397,7 @@ export default {
               }
               editor.commands.first(({ commands }) => [
                 () => commands.newlineInCode(),
-                () => commands.splitListItem('listItem'),
+                () => self.allowBulletList ? commands.splitListItem('listItem') : false,
```

Comment:
Does this suggestion work? I like it more.

Or is it necessary to return that false?

```suggestion
                () => self.allowBulletList && commands.splitListItem('listItem'),
```

Reply:
Should work fine it's the exact same logic. Still returns false if allowBulletList is false.